src/components/Book.js

```diff
@@ -1,27 +1,58 @@
 import PropTypes from 'prop-types';
 import { useDispatch } from 'react-redux';
 import { deleteBook } from '../redux/books/booksSlice';
+import './Book.css';
 
 const Book = ({ book }) => {
   const dispatch = useDispatch();
   const onRemoveClicked = (id) => {
     dispatch(deleteBook(id));
   };
   return (
-    <div>
-      <h1>{book.title}</h1>
-      <h2>
-        Author:
-        {book.author}
-      </h2>
-      <button onClick={() => onRemoveClicked(book.item_id)} type="button">Delete</button>
+    <div className="main-book" key={book.item_id}>
+      <div>
+        <p className="category">{book.category}</p>
+        <h1 className="Title">{book.title}</h1>
+        <p className="author">
+          Author:
+          {' '}
+          {book.author}
+        </p>
```

Comment:
- [ ] You're making great progress in aligning your project with the design, and I appreciate your dedication to achieving a design-compliant user interface. As mentioned earlier, it's important to update your code to match the font styles, colors, and gaps outlined in the design. By making these adjustments, you'll ensure that your project closely aligns with the intended design, creating a more visually consistent and polished user interface.

- [ ] In addition to these styling improvements, I noticed that you're displaying "Author: " before each author's name, which is not present in the design. To better match the design, please update your code to remove the "Author: " prefix.

Keep up the excellent work, and thank you for your commitment to adhering to the design guidelines! 🌟🎨